client/verta/docs/tutorials/deployment/endpoint_creation.rst

```diff
@@ -0,0 +1,35 @@
+Creating and Retrieving an Endpoint using the Client and CLI
+============================================================
+
+Verta's Endpoint provides an API for configurable deployment of machine learning models.
+
+This tutorial will go through how to create and retrieve such an endpoint using Verta Client and CLI.
+
+Using the Client
+----------------
+
+Users can create an Endpoint as follows:
+
+.. code-block:: python
+
+    endpoint = client.create_endpoint(path="/some-path")
+
+This Endpoint can be retrieved later on via its ID or its path:
```

Comment:
Make sure the wording is the same order as the examples, to reduce confusion
```suggestion
This Endpoint can be retrieved later on via its path or its ID:
```

Reply:
Oops. Fixed.